### PR TITLE
New status messages format

### PR DESCRIPTION
**Pull request by polypiel**

### CODE REVIEW

File: src/data-parser.js [hunk 1]
```diff
@@ -105,6 +105,7 @@ const getMessage = inputMessage => {
                 simInfo = identifySim({
                     message: inputMessage,
                     simData: simData,
+                    hlChar: '*',
```
Comment (Winde) [inline src/data-parser.js]: is hlChar name significant?

File: src/data-parser.js [hunk 2]
```diff
@@ -55,44 +55,44 @@ const getSimDataFromMessage = message => {
...
 
 const serializeMessageId = message =>
     (message.msisdn && message.msisdn != '?') ||
     (message.channel != undefined && 'Channel ' + message.channel) ||
     '';
 
-const identifySim = ({message, simData}) =>
-    serializeSimData(simData) ? serializeSimData(simData) : serializeMessageId(message);
+const identifySim = ({message, simData, hlChar}) =>
+    serializeSimData({simData, hlChar}) ? serializeSimData({simData, hlChar}) : serializeMessageId(message);
```
Comment (Winde) [inline src/data-parser.js]: Should hlChar come from outside but be part of serialization here?

File: src/data-parser.js (hunk 1 above)
Comment (polypiel) [reply src/data-parser.js]: Not very indeed. It comes from hightlightChar, I'll change it to hightlightChar

File: src/data-parser.js (hunk 2 above)
Comment (polypiel) [reply src/data-parser.js]: I don't follow you, what do you mean?

**Pull request closed without merging**